counterparty-core/counterpartycore/test/integrations/regtest/properytestnode.py

```diff
@@ -1,9 +1,17 @@
+import json
```

Comment:
fyi there's a typo in this file name (propery -> property)

Reply:
thanks! fixed